k9mail/src/main/java/com/fsck/k9/preferences/SettingsImporter.java

```diff
@@ -133,7 +134,13 @@ public static ImportContents getImportStreamContents(InputStream inputStream)
             // will not be null.
             if (imported.accounts != null) {
                 for (ImportedAccount account : imported.accounts.values()) {
-                    accounts.add(new AccountDescription(account.name, account.uuid));
+                    //Show a alternative text in the case the description is empty
+                    String name = account.name;
+                    if (TextUtils.isEmpty(name) && account.identities != null && account.identities.size() > 0){
```

Comment:
We have code similar to this in a couple of places. It would probably make more sense to introduce an `AccountNamePrinter` class so the output is consistent and we don't have duplicate code all over the place.